lib/rules/no-down-event-binding.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const { match } = require('../helpers/node-matcher');
+const Rule = require('./base');
+
+const ERROR_MESSAGE = 'Avoid binding to a `down` event; bind to an `up` event instead';
+
+/**
+ * Detects that a Node is an instance of the `{{on}}` modifier with a known event that is being listened to
+ *
+ * @param {object} node
+ * @return {boolean}
+ */
+function isOnModifier(node) {
+  return match(node, { path: { original: 'on' }, params: [{ type: 'StringLiteral' }] });
+}
+
+/**
+ * Detects that a Node is an instance of the `{{action}}` modifier
+ *
+ * @param {object} node
+ * @return {boolean}
+ */
+function isActionModifer(node) {
+  return match(node, { path: { original: 'action' } });
+}
+
+/**
+ * Check if an event name is a "down" event
+ *
+ * @param {string} eventName
+ * @return {boolean}
+ */
+function isDownEvent(eventName) {
+  return eventName.toLowerCase().endsWith('down');
+}
+
+module.exports = class NoDownEventBinding extends Rule {
+  visitor() {
+    return {
+      ElementModifierStatement(node) {
+        if (isOnModifier(node)) {
+          const eventNameNode = node.params[0];
+
```

Comment:
Could we have fixer for this rule? 
Looks like it’s a simple one

Reply:
It would definitely be easy to write! My only concern is the idea of auto-fixing something that is really different behavior. I don't know what I would want to fix this on save and not notice that things had changed, in case you really *do* need to bind to a "down" event..

Reply:
Autofixers should not change behavior. Changing the event used could be a significant change in behavior.